tests/test_cpio.py

```diff
@@ -114,10 +114,19 @@ def archiveCreate(self, fn, fmt='w'):
     def doArchive(self, fn, fmt=None):
         self.archiveExtract(fn)
         fn2 = "archive2" + fn[len("archive"):]
+        # Diff-Coverage testing enforcement in GitHub CI revealed:
```

Comment:
This looks like the kind of thing that is better in the commit message (which it already is) rather than a comment.

Reply:
Thanks, agreed, done!